internal/npm_install/generate_pnp_build_files.ts

```diff
@@ -0,0 +1,47 @@
+import { promises as fsp, existsSync, mkdirSync } from "fs";
+import * as path from "path";
+
+function writeFileSync(p: string, content: string) {
+  if (!existsSync(path.dirname(p))) {
+    mkdirSync(path.dirname(p), { recursive: true });
+  }
+  return fsp.writeFile(p, content);
+}
+
+function createBuildFile(packagePath: string) {
+  const contents = `package(default_visibility = ["//visibility:public"])
+load("@build_bazel_rules_nodejs//internal/npm_install:node_module_library.bzl", "node_module_library")
+
+filegroup(
+    name = "${packagePath}",
+    srcs = [] # we dont care about the actual content, we want a list of dependencies to modify the pnp file around
+)
+`;
+
+  return writeFileSync(`${packagePath}/BUILD.bazel`, contents);
+}
+
+async function main(workspacePath: string) {
+  const contents = `package(default_visibility = ["//visibility:public"])
+exports_files([
+    ".pnp.js",
+    "package.json",
+])`;
+  const packageJson = require(path.join(workspacePath, "package.json"));
+  await Promise.all(
+    [
+      "dependencies",
+      "devDependencies",
+      "peerDependencies",
+      "optionalDependencies"
+    ].map(d => {
+      Promise.all(
+        Object.keys(packageJson[d] || {})
+          .map(createBuildFile)
+          .concat([writeFileSync(`BUILD.bazel`, contents)])
```

Comment:
```suggestion
          .concat([writeFileSync("BUILD.bazel", contents)])
```
or
```suggestion
          .concat([writeFileSync('BUILD.bazel', contents)])
```

as you're not using any interpolation.